CHANGELOG.md

```diff
@@ -1,5 +1,7 @@
 ## main / unreleased
 
+* [BUGFIX] Correctly publish the hedgedmetrics counter with delta [#4078](https://github.com/grafana/tempo/pull/4078) (@electron0zero)
+* [BUGFIX] Replace hedged requests roundtrips total with a counter. [#4063](https://github.com/grafana/tempo/pull/4063) (@galalen)
```

Comment:
this entry was in `# v2.6.0` section, should be in `## main / unreleased` so fixed it.